datafusion/sql/tests/sql_integration.rs

```diff
@@ -161,70 +160,6 @@ fn parse_ident_normalization() {
     }
 }
 
-#[test]
-fn test_parse_options_value_normalization() {
```

Comment:
Could you please put this test back and updated it so that it passes? That way we can reason about / more easily see the change in behavior introduced by this PR